### ISSUE TITLE
[Question] The optimizer used for training

### ISSUE THREAD

**Issue by fahos** [princeton-nlp/SimCSE#287]
Thank you for providing the source code.
Is the optimizer used to train the SimCSE model published on the huggingface hub AdamW?
Could you please provide more information about the optimizer used during training?
Thank you.

**Comment by gaotianyu1350** [comment]
Yes it is AdamW.

**Comment by fahos** [comment]
Thank you for your quick response.
Are each parameter of AdamW specified besides the learning rate the default values for transformers' AdamW?
https://huggingface.co/docs/transformers/v4.46.0/en/main_classes/optimizer_schedules#transformers.AdamW

**Comment by gaotianyu1350** [comment]
Hi, yes, and you should be able to reproduce our results with the scripts provided in this repo.

**Comment by fahos** [comment]
Thank you for answering my question. I really appreciate it.
